src/main/scala/Vcs.scala

```diff
@@ -201,6 +212,11 @@ class Subversion(val baseDir: File) extends Vcs {
 
   override def currentBranch: String = workingDirSvnUrl.substring(workingDirSvnUrl.lastIndexOf("/") + 1)
 
+  // FIXME: Need help here
+  def setBranch(branch: String) = cmd("checkout", branch)
+
+  def newBranch(branch: String) = cmd("checkout", "-b", branch)
+
```

Comment:
This is definitely not the right behaviour for svn, I think svn branching requires doing an `svn copy` from trunk to `/branches/<branch-name>`.  I'll be happy to merge if, rather than doing the wrong thing, these throw an exception to say this operation is not yet supported by svn.